### ISSUE TITLE
Configuration for rule "jsx-a11y/label-has-for" is invalid: Value "label" should be object. 

### ISSUE THREAD

**Issue by realtebo** [evcohen/eslint-plugin-jsx-a11y#348]
```
c:\Users\realtebo\Downloads\manager\node_modules\eslint-config-rallycoding\rules\react-a11y.js: Configuration for rule "jsx-a11y/label-has-for" is invalid: Value "label" should be object. 

Referenced from: c:\Users\realtebo\Downloads\manager\node_modules\eslint-config-rallycoding\index.js 
Referenced from: c:\Users\realtebo\Downloads\manager\.eslintrc

```
My acutal package.json

```
  "devDependencies": {
    "eslint": "4.9.0",
    "eslint-config-rallycoding": "^3.2.0",
    "eslint-plugin-import": "2.7.0",
    "eslint-plugin-jsx-a11y": "6.0.2",
    "eslint-plugin-react": "7.4.0",
```

**Comment by ljharb** [comment]
What's in your .eslintrc?

**Comment by realtebo** [comment]
I simple extended eslint-rallycoding

**Comment by ljharb** [comment]
It looks like rallycoding is a fork of Airbnb’s config; it’s only recently that airbnb’s config supports v6 of this plugin. I’d file the issue there.

**Comment by realtebo** [comment]
Thanks. While waiting, I manually rollbacked to an older version.

**Comment by Iippa** [comment]
I'm having the same problem. How do I manually rollback?

**Comment by Iippa** [comment]
I have the same problem. How do I rollback to an older version?

**Comment by dakdreams** [comment]
add this lime in eslintrc
`"rules": {
    "jsx-a11y/label-has-for": false
    }`

**Comment by nicoladj77** [comment]
@dakdreams I added it but stil get an error:

```
{
  "extends": "rallycoding",
  "rules": {
    "jsx-a11y/label-has-for": false
  }
}
```

> nicola@Questions$ eslint src/**
> /usr/local/lib/node_modules/eslint-config-rallycoding/rules/react-a11y.js:
> 	Configuration for rule "jsx-a11y/label-has-for" is invalid:
> 	Value ["label"] should be object.
> 
> Referenced from: /usr/local/lib/node_modules/eslint-config-rallycoding/index.js
> Referenced from: /Users/nicola/Questions/.eslintrc
> Error: /usr/local/lib/node_modules/eslint-config-rallycoding/rules/react-a11y.js:
> 	Configuration for rule "jsx-a11y/label-has-for" is invalid:
> 	Value ["label"] should be object.
> 
> Referenced from: /usr/local/lib/node_modules/eslint-config-rallycoding/index.js
> Referenced from: /Users/nicola/Questions/.eslintrc
>     at validateRuleOptions (/usr/local/lib/node_modules/eslint/lib/config/config-validator.js:119:19)
>     at Object.keys.forEach.id (/usr/local/lib/node_modules/eslint/lib/config/config-validator.js:162:9)
>     at Array.forEach (<anonymous>)
>     at validateRules (/usr/local/lib/node_modules/eslint/lib/config/config-validator.js:161:30)
>     at Object.validate (/usr/local/lib/node_modules/eslint/lib/config/config-validator.js:239:5)
>     at loadFromDisk (/usr/local/lib/node_modules/eslint/lib/config/config-file.js:507:19)
>     at load (/usr/local/lib/node_modules/eslint/lib/config/config-file.js:550:20)
>     at configExtends.reduceRight (/usr/local/lib/node_modules/eslint/lib/config/config-file.js:421:36)
>     at Array.reduceRight (<anonymous>)
>     at applyExtends (/usr/local/lib/node_modules/eslint/lib/config/config-file.js:403:28)
> (node:52429) DeprecationWarning: [eslint] The 'ecmaFeatures' config file property is deprecated, and has no effect. (found in /usr/local/lib/node_modules/eslint-config-rallycoding/rules/react.js)
> (node:52429) DeprecationWarning: [eslint] The 'ecmaFeatures' config file property is deprecated, and has no effect. (found in /usr/local/lib/node_modules/eslint-config-rallycoding/rules/react-a11y.js)

**Comment by ljharb** [comment]
`false` isn’t a valid value there; try `0` or `’off’`

**Comment by jonathanendy** [comment]
Hi All,
I got the same issue and I'm trying to debug it, I'm not a JS expert nor Atom but these are my findings.
It seems to me the if statement continues with the null value, what am I missing?
Please look at the picture
[https://ibb.co/hFGbac](url)
